internal/services/storagecache/managed_lustre_file_system_auto_export_job_resource_test.go

```diff
@@ -0,0 +1,95 @@
+package storagecache_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/go-azure-helpers/lang/pointer"
+	"github.com/hashicorp/go-azure-helpers/lang/response"
+	"github.com/hashicorp/go-azure-sdk/resource-manager/storagecache/2024-07-01/autoexportjobs"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/acceptance/check"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type ManagedLustreFileSystemAutoExportJobResource struct{}
+
+func (r ManagedLustreFileSystemAutoExportJobResource) Exists(ctx context.Context, clients *clients.Client, state *pluginsdk.InstanceState) (*bool, error) {
+	id, err := autoexportjobs.ParseAutoExportJobID(state.ID)
```

Comment:
Can we add some more tests here, at least a `requiresImport` and an `update`?

Reply:
Will do.